charts/libsql/values.yaml

```
@@ -0,0 +1,79 @@
+
+# This section is for setting up autoscaling more information can be found here: https://kubernetes.io/docs/concepts/workloads/autoscaling/
+autoscaling:
+  enabled: false
+  minReplicas: 1
+  maxReplicas: 100
+  targetCPUUtilizationPercentage: 75
+  targetMemoryUtilizationPercentage: 75
+
+# Additional volumes on the output Deployment definition.
+volumes:
+ - name: data
```

YAMLlint (1.37.1)

[warning] wrong indentation: expected 0 but found 1

(indentation)